cmake/std/atdm/shiller/environment.sh

```diff
@@ -97,7 +101,7 @@ export ATDM_CONFIG_HDF5_LIBS="-L${HDF5_ROOT}/lib;-lhdf5_hl;-lhdf5;-lz;-ldl"
 export ATDM_CONFIG_NETCDF_LIBS="-L${BOOST_ROOT}/lib;-L${NETCDF_ROOT}/lib;-L${NETCDF_ROOT}/lib;-L${PNETCDF_ROOT}/lib;${BOOST_ROOT}/lib/libboost_program_options.a;${BOOST_ROOT}/lib/libboost_system.a;${NETCDF_ROOT}/lib/libnetcdf.a;${PNETCDF_ROOT}/lib/libpnetcdf.a;${ATDM_CONFIG_HDF5_LIBS}"
 
 # Use manually installed cmake and ninja (see TRIL-209)
-export PATH=/home/rabartl/install/hansen-shiller/cmake-3.11.2/bin:/home/rabartl/install/hansen-shiller/ninja-1.8.2/bin:$PATH
+export PATH=/ascldap/users/rabartl/install/hansen-shiller/cmake-3.11.2/bin:/ascldap/users/rabartl/install/hansen-shiller/ninja-1.8.2/bin:$PATH
```

Comment:
I think we need to change this back to `/home/rabartl/`.  That directory path `/ascldap/users/rabartl/` does not exist on the compute nodes as shown by:

```
[rabartl@shiller01 ~ (master)]$ salloc -N 1
salloc: Granted job allocation 1001233

bash-4.1$ hostname
shiller03

bash-4.1$ ls /ascldap/users/rabartl/install/hansen-shiller/
ls: cannot access /ascldap/users/rabartl/install/hansen-shiller/: No such file or directory

bash-4.1$ ls /home/rabartl/install/hansen-shiller/
cmake-3.11.2  ninja-1.8.2
```